earth_enterprise/src/scons/khEnvironment.py

```diff
@@ -141,6 +142,31 @@ def WriteToFileStrfunc(file_name, strn):
   return 'WriteToFile(%s, %s)' % (file_name, strn)
 
 
+def StringExpandFileFunc(target, source, env):
```

Comment:
we are creating a file and saving env variables into ? what is the reason ?

Reply:
We're not saving environment variables in a file. This function expands variables in a template file.  The reason is in `cgi.conf.in`.  I think this is better than using `sed` from the command line to substitute substrings in a file as is done in other `scons` targets.

Reply:
ok